src/main/java/de/tum/in/www1/artemis/web/rest/util/HeaderUtil.java

```diff
@@ -19,23 +19,23 @@ private HeaderUtil() {
     }
 
     public static HttpHeaders createAlert(String applicationName, String message, String param) {
-        return io.github.jhipster.web.util.HeaderUtil.createAlert(applicationName, message, param);
+        return tech.jhipster.web.util.HeaderUtil.createAlert(applicationName, message, param);
     }
 
     public static HttpHeaders createEntityCreationAlert(String applicationName, boolean enableTranslation, String entityName, String param) {
-        return io.github.jhipster.web.util.HeaderUtil.createEntityCreationAlert(applicationName, enableTranslation, entityName, param);
+        return tech.jhipster.web.util.HeaderUtil.createEntityCreationAlert(applicationName, enableTranslation, entityName, param);
     }
 
     public static HttpHeaders createEntityUpdateAlert(String applicationName, boolean enableTranslation, String entityName, String param) {
-        return io.github.jhipster.web.util.HeaderUtil.createEntityUpdateAlert(applicationName, enableTranslation, entityName, param);
+        return tech.jhipster.web.util.HeaderUtil.createEntityUpdateAlert(applicationName, enableTranslation, entityName, param);
     }
 
     public static HttpHeaders createEntityDeletionAlert(String applicationName, boolean enableTranslation, String entityName, String param) {
-        return io.github.jhipster.web.util.HeaderUtil.createEntityDeletionAlert(applicationName, enableTranslation, entityName, param);
+        return tech.jhipster.web.util.HeaderUtil.createEntityDeletionAlert(applicationName, enableTranslation, entityName, param);
     }
 
     public static HttpHeaders createFailureAlert(String applicationName, boolean enableTranslation, String entityName, String errorKey, String defaultMessage) {
-        HttpHeaders headers = io.github.jhipster.web.util.HeaderUtil.createFailureAlert(applicationName, enableTranslation, entityName, errorKey, defaultMessage);
+        HttpHeaders headers = tech.jhipster.web.util.HeaderUtil.createFailureAlert(applicationName, enableTranslation, entityName, errorKey, defaultMessage);
```

Comment:
Can you replace the package calls with imports?

Reply:
I'd assume that would clash with the class name

Reply:
Ah, good point. I missed that.